tools/swift-inspect/README.md

```diff
@@ -16,6 +16,14 @@ In order to build on Windows, some additional parameters must be passed to the b
 swift build -Xcc -I%SDKROOT%\usr\include\swift\SwiftRemoteMirror -Xlinker %SDKROOT%\usr\lib\swift\windows\x86_64\swiftRemoteMirror.lib
 ~~~
 
+#### Linux
+
+In order to build on Linux, some additional parameters must be passed to the build tool to loace the necessary includes and libraries
+
+~~~
+swift build -Xswiftc -I$(git rev-parse --show-toplevel)/include/swift/SwiftRemoteMirror -Xswiftc -lswiftRemoteMirror
```

Comment:
Can we use `systemLibrary` to avoid the `-Xswiftc -lswiftRemoteMirror`? I think that we should also avoid the `-Xswiftc -lswiftRemoteMirror` at least in favour of `-Xlinker -lswiftRemoteMirror` or `-Xclang-linker -lswiftRemoteMirror`.

Reply:
This should definitely be `-Xlinker -lswiftRemoteMirror` instead of using `-Xswiftc`.

I don't see how I can add a dependency on `SwiftRemoteMirror` as a `systemLibrary` since it is in a different location of the source tree.